### PR TITLE
Section component and dashboard layout

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/21298

*Description of changes:*
Add the section component and set up dashboard layout

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-7-slwvqg5x.web.app

**Review by kialj876 — APPROVED** [review]

File: src/components/bcros/Section.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div :data-cy="'section_' + name">
+    <div :data-cy="'header_' + name" class="font-bold text-lg py-2">
```
Comment (kialj876) [inline src/components/bcros/Section.vue:3]: I think it would be better to remove the top padding here and let us apply that where we use this component

Comment (kialj876) [reply src/components/bcros/Section.vue:3]: otherwise we end up with scenarios where we need to apply negative or layered padding on top of this component

Comment (patrickpeinanw) [reply src/components/bcros/Section.vue:3]: I will update this before merging the PR

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-7-slwvqg5x.web.app

**Pull request merged**